traits/trait_list_object.py

```diff
@@ -18,30 +19,27 @@
 class TraitListEvent(object):
     """ An object reporting in-place changes to a traits list.
 
-    Parameters
-    ----------
-    index : int or slice
-        An index or slice indicating the location of the changes to the list.
-    added : list or None
-        The list of values added to the list, or optionally None if nothing
-        is added.
-    removed : list or None
-        The list of values removed from the list, or optionally None if
-        nothing is removed.
-
     Attributes
     ----------
     index : int or slice
-        An index or slice indicating the location of the changes to the list.
+        The index of the change in the list.
     added : list
-        The list of values added to the list.  If nothing was added this is
-        an empty list.
+        The list of values added to the list.
     removed : list
-        The list of values removed from the list.  If nothing was removed
-        this is an empty list.
+        The list of values removed from the list.
     """
 
     def __init__(self, index=0, removed=None, added=None):
+        """
```

Comment:
The way that the TraitsUI documentation is built, `__init__` docstrings aren't picked up, only the class docstrings, so `__init__` parameters are expected to be described in the class docstring.  I'd suggest reverting the change.